### PR TITLE
New Entry: R loops

### PR DESCRIPTION
**Pull request by yangc95**
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/r/concepts/loops/loops.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Tags:
+  - 'Control Flow'
+  - 'Iterators'
+  - 'Loops'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+A loop is used to perform some operation given that a declared condition is truthy. In R, there are `while` and `for` loops.
```
Comment (Dusch4593) [inline content/r/concepts/loops/loops.md:16]:
```suggestion
A loop is used to perform some operation as long as declared condition is truthy. In R, there are `while` and `for` loops.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yangc95 Finished my review and left some comments/suggestions.

**Review by yangc95 — COMMENTED** [review]
@Dusch4593 implemented your suggestions, TY!

**Comment by KTom101** [comment]
@yangc95 Is this ready for a second review? I see a few unresolved changes.

**Comment by yangc95** [comment]
@KTom101 thanks for letting me know! I thought I applied the suggestions but it looks like it wasn't done successfully - just updated 🙌

**Review by Dusch4593 — APPROVED** [review]
@yangc95 LGTM for the next review 👍🏻

File: content/r/concepts/loops/loops.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Loops'
+Description: 'A loop is used to perform some operation as long as a declared condition is truthy.'
```
Comment (SSwiniarski) [inline content/r/concepts/loops/loops.md:3]:
```suggestion
Description: 'A loop is used to repeat some operation a multiple number of times.'
```

Comment (SSwiniarski) [reply content/r/concepts/loops/loops.md:3]: The original description applied to `while` loops, but not `for` loops.

File: content/r/concepts/loops/loops.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Tags:
+  - 'Control Flow'
+  - 'Iterators'
+  - 'Loops'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+A loop is used to perform some operation as long as a declared condition is truthy. In R, there are `while` and `for` loops.
```
Comment (SSwiniarski) [inline content/r/concepts/loops/loops.md:16]:
```suggestion
A loop is used to repeat some operation a multiple number of times. In R, there are `while` and `for` loops.
```

Comment (SSwiniarski) [reply content/r/concepts/loops/loops.md:16]: Again, it needs to apply to `for` as well as `while` loops.

File: content/r/concepts/loops/loops.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## While Loop
+
+The statement(s) within a `while` loop code block will continue to execute given that a declared condition is `TRUE`.
```
Comment (SSwiniarski) [inline content/r/concepts/loops/loops.md:20]:
```suggestion
The statement(s) within a `while` loop code block will continue to execute as long as a declared condition is `TRUE`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@yangc95 I completed a second review and left a few comments.

**Comment by yangc95** [comment]
@SSwiniarski thanks for those logical change! I've implemented your suggestions

**Review by SSwiniarski — APPROVED** [review]
@yangc95 Looks good!

**Comment by Dusch4593** [comment]
This looks ready to merge! 🚀

I went and bolded the concept "loop" in 87dc9fe6dc2e03341da9d85c8dae3ee7415850fb.

cc: @yangc95 @SSwiniarski

**Pull request merged**